.vale/styles/plain-language-guidelines/be-consise.yml

```diff
@@ -0,0 +1,28 @@
+# Vale-code for the lists on https://www.plainlanguage.gov/guidelines/concise/
```

Comment:
We can package our Plain language guidelines Vale code into a `.zip` file that's attached to a GitHub Release. That's probably the easiest way to deal with the public domain license and our different license.

It will be harder for me though, as now I'll need to figure out how to package and version a `.zip` file in a new repository.

It would work something like this:

1. Create a new repository: `honkinggoose/vale-plain-language-guidelines`
2. Put Vale code into that repository
3. Package Vale code into a `.zip`
4. In the Renovate repositories that use Vale, we should then load the `.zip` file
5. Update our Renovate bot config so it can update the `.zip` file if the upstream repo releases a new `.zip`

Documentation for packaging: https://vale.sh/docs/topics/packages/

Reply:
What's the need to package the guidelines? Is it purely a licensing issue, or the need to share common configuration across multiple repositories?

Reply:
Renovate is not bundling/repackaging value in any way, so I think having a separate lint action makes even more sense if you have a concern here

Reply:
> What's the need to package the guidelines? Is it purely a licensing issue,

@JamieMagee I don't know if you can put Public Domain stuff into a AGPL project. The Plain Language guidelines are _Public Domain_, while Renovate is _AGPL_.

Also, by packaging the guidelines we can share the guidelines with our other repositories and with other people.

Reply:
From some quick research, it seems like there is no issue with license compatibility. An excerpt from GNU[^1]:

> Being in the public domain is not a license; rather, it means the material is not copyrighted and no license is needed. Practically speaking, though, if a work is in the public domain, it might as well have an all-permissive non-copyleft free software license. **Public domain material is compatible with the GNU GPL**.

There is also a section on the Wikipedia page for the GPL about license compatibility[^2] which uses the Free Software Foundation as a source to explicitly show the licenses as being compatible.

[^1]: https://www.gnu.org/licenses/license-list.en.html
[^2]: https://en.wikipedia.org/wiki/GNU_General_Public_License#Compatibility_and_multi-licensing